app/views/layout.html

```diff
@@ -60,6 +59,31 @@
         }
       ] %}
 
+{% set sdtChangeLog1 %}
```

Comment:
Could this be a dictionary/map of version numbers and where to find them, which could be looped over below? Saves having to add it in 2 places for each version.